CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
```

Comment:
```suggestion
### Removed
```
Also, I don't think there's such a thing as "Conjur v5.0.0+". We can say "We recommend users migrate to Dynamic Access Provider v11+ or Conjur OSS v1+"

Reply:
Definitely wasn't sure what to put here - thank you!